config/routes.rb

```diff
@@ -48,6 +48,9 @@
       get   :accept
       patch :accept_assignment
       patch :accept_invitation
+      get   :setupv2
+      post  :create_repo
+      get   :progress
```

Comment:
Why are there 3 new routes here, but only one new view/test file?

Reply:
Ah, I just reviewed the other PR and saw tests and controller actions defined there. I'd either clean up the diff or merge them together, merging this without the controller actions / views from the other PR would result in broken paths (that probably wouldn't be hit, but best to be tidy and keep things together.